City-Bureau/city-scrapers-akr
# 🏗️ Build spider: Akron City Council - Committees

## What's this PR do?
Tweaks the Akron City Council spider to only scraper meetings for the city council itself and creates a separate spider to scrape committee meetings called `akr_city_council_committees`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl akr_city_council_committees -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here]().

## Are there any smells or added technical debt to note?


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207060091579318
